.cookiecutter/includes/tox/passenv

```diff
@@ -11,7 +11,10 @@ dev: NEW_RELIC_APP_NAME
 dev: NODE_ENV
 dev: PROXY_AUTH
 dev: OIDC_AUTHORIZATIONURL_GOOGLE
+dev: OIDC_AUTHORIZATIONURL_FACEBOOK
```

Comment:
As with Google you _can_ set these envvars to override the Facebook URLs in dev if you want, hence these are added to tox's `passenv`, but there are hard-coded defaults for them so we don't actually need to set these envvars in dev, staging or prod.